src/renderer/services/file.js

```diff
@@ -338,7 +339,12 @@ class FileProvider {
       return Promise.reject(new Error('File does not exist'))
     }
     const timestamp = fileHelper.getUtcTimestamp(file)
-    const songMeterFileInfo = await new SongMeterFileInfo(file.path)
+    const isWavFile = file.extension === 'wav'
+    let convertedPath
+    if (!isWavFile) {
+      convertedPath = (await fileHelper.convert(file.path, remote.app.getPath('temp'), file.streamId, null, 'wav')).path
+    }
+    const songMeterFileInfo = await new SongMeterFileInfo(isWavFile ? file.path : convertedPath)
```

Comment:
Remove this... if the file is not `wav` then do not convert it to `wav`.

Reply:
Better would be to check if it is wav before calling SongMeterFileInfo. e.g. one line change

```
const songMeterFileInfo = file.extension === 'wav' ? await new SongMeterFileInfo(file.path) : { metadata: '' }
```